tree_assume/src/schedule.egg

```diff
@@ -4,5 +4,8 @@
     (repeat 100 assume)
     (saturate always-run)
     (saturate error-checking)
-    (saturate type-analysis)
+    (repeat 10 ; TODO: do this til saturation?
```

Comment:
We discussed that this can be a saturate
I would use a different name than "immediate"
Maybe type-helpers?